### PR TITLE
add-position-saved

### PR DESCRIPTION
**Pull request by pratikshamandlik99**
Add Position

### CODE REVIEW

File: ats-web/src/api/positionApi.js [hunk 1]
```diff
@@ -1,13 +1,29 @@
 import { handleResponse, handleError } from "./apiUtils";
 // const baseUrl = process.env.REACT_APP_API_URL + "/positions/";
-const basePortUrl = "http://13.233.58.211:7000";
-const baseUrl = basePortUrl + "/positions";
+
+const baseUrl = "http://13.233.58.211:7000/positions";
+const proj_baseUrl = "http://13.233.58.211:7000/projects";
```
Comment (rakhimundhada15) [inline ats-web/src/api/positionApi.js]: We all ready have common method (getDetails)to fetch data from  port 7000 in shared api. Just pass  endpoint(ex. '/positions)

File: ats-web/src/api/positionApi.js [hunk 2]
```diff
@@ -1,13 +1,29 @@
...
+const emp_baseUrl = "http://13.233.58.211:7000/employees";
+
+
 export function getPositions() {
   return fetch(baseUrl)
     .then(handleResponse)
     .catch(handleError);
 }
 
+export function getProjects() {
```
Comment (rakhimundhada15) [inline ats-web/src/api/positionApi.js]: Use method (getDetails) from shared api .

File: ats-web/src/api/positionApi.js [hunk 3]
```diff
@@ -1,13 +1,29 @@
...
+  return fetch(proj_baseUrl)
+    .then(handleResponse)
+    .catch(handleError);
+}
+
+export function getEmployees() {
```
Comment (rakhimundhada15) [inline ats-web/src/api/positionApi.js]: Use method (getDetails) from shared api .

**Comment by sameerbag** [comment]
Branch is not updated with master.

**Comment by pratikshamandlik99** [comment]
closing will create new one

**Pull request closed without merging**